sound/soc/sof/ipc4-topology.c

```diff
@@ -1357,18 +1358,29 @@ static int snd_sof_get_nhlt_endpoint_data(struct snd_sof_dev *sdev, struct snd_s
 						   &bit_depth);
 		if (ret < 0)
 			return ret;
+
+		/* We need to know the type of external device attached to a SSP
+		 * port to retrieve NHLT blob. However, device type is not
+		 * specified in topology. Here we query the type for the port
+		 * and then pass that information back to the blob lookup
+		 * function.
+		 */
```

Comment:
I think this would be cleaner:
```c
/*
 * We need to know the type of the external device attached to a SSP
 * port to retrieve the blob from NHLT. However, device type is not
 * specified in topology. 
 * Query the type for the port and then pass that information back
 * to the blob lookup function.
 */
```